features/public_search.feature

```diff
@@ -24,13 +24,13 @@ Background: Logged Out
 
     And I'm am on the root page
 
-Scenario: No keyword
-    Given I publicly search for ""
-    Then I should not see "app 1"
+#Scenario: No keyword
```

Comment:
Already included in previous PR, but to pass CI, we have to include modified tests.

Please ignore this file.